src/TRE-API/appsettings.Development.json

```diff
@@ -113,6 +113,9 @@
   "TreAPISettings": {
     "Address": "https://localhost:7157"
   },
+  "TreUISettings": {
+    "Address": "https://localhost:7187"
```

Comment:
Why does the TRE Agent API now need to know the TRE Agent UI address? (I guess CORS or something?)

but there is no change in the PR using this value?